internal/ent/privacy/rule/features.go

```diff
@@ -131,20 +136,48 @@ func AllowIfHasFeature(feature string) privacy.QueryMutationRule {
 }
 
 // HasAnyFeature checks if any of the provided features are enabled for the organization
-func HasAnyFeature(ctx context.Context, feats ...string) (bool, error) {
+func HasAnyFeature(ctx context.Context, feats ...models.OrgModule) (bool, error) {
+	return checkFeatures(ctx, false, feats...)
+}
+
+// HasAllFeatures checks if all of the provided features are enabled for the organization
+func HasAllFeatures(ctx context.Context, feats ...models.OrgModule) (bool, error) {
+	return checkFeatures(ctx, true, feats...)
+}
+
+// checkFeatures is a utility function that checks features based on the requireAll flag
+// If requireAll is true, all features must be enabled.
+//
+// If false, at least one must be enabled.
+func checkFeatures(ctx context.Context, requireAll bool, feats ...models.OrgModule) (bool, error) {
 	enabled, err := orgFeatures(ctx)
 	if err != nil {
 		return false, err
 	}
 
+	if len(enabled) == 0 {
+		return true, nil
+	}
+
 	enabledSet := make(map[string]struct{}, len(enabled))
 
 	for _, f := range enabled {
 		enabledSet[f] = struct{}{}
 	}
 
+	if requireAll {
+		// all features must be enabled
+		for _, f := range feats {
+			if _, ok := enabledSet[string(f)]; !ok {
```

Comment:
OrgModule has a `String()` function 
```suggestion
			if _, ok := enabledSet[f.String()]; !ok {
```